packages/dev/test/RelayHubPenalizations.test.ts

```diff
@@ -481,7 +481,7 @@ contract('RelayHub Penalizations', function ([_, relayOwner, committer, nonCommi
           const method = await commitPenalizationAndReturnMethod(
             penalizer.contract.methods.penalizeRepeatedNonce, txDataSigA.data, txDataSigA.signature, txDataSigB.data, txDataSigB.signature, relayHub.address)
           await expectRevert(
-            method.send({ from: reporterRelayManager }),
+            method.send({ from: reporterRelayManager, gas: 5e5 }),
```

Comment:
forward looking: EIP1559's support in `hardhat node` as minimum basefee of 8 gwei, so if you specify gas, at least put a meaningful number like 10gwei, to avoid changing it again...)